### ISSUE TITLE
Send email to Tim Dallman regarding QC for Salmonella at PHE

### ISSUE THREAD

**Issue by Camilsek** [NorwegianVeterinaryInstitute/IRIDA_Plugin_Development_Steps#23]
Camilla will send an email to Tim Dallman

**Comment by Camilsek** [comment]
Hi Camilla.
Sorry for the delay in replying been deep in grant submission.
The PHE pipeline is (was!?) exclusively read-based so all QC is done on that.
Basically the steps are:

•	>30x coverage (~150mb) after quality trimming with Trimmomatic (leading and trailing > PHRED 30) discarding read pairs < 50bp
•	Single (and correct!) species identified in reads using kmerID (https://github.com/phe-bioinformatics/kmerid) – this might have been replaced with Kraken
•	A series of metrics associated with MLST calling using MOST (https://github.com/phe-bioinformatics/most) – i.e. max non-consensus bases
•	A series of quality metrics based of reference SNP mapping by SnapperDB.

Basically the QC comes down:
•	Do I have enough high quality data?
•	Is my data contaminated (with another species)?
•	Is my sample mixed (with another MLST/serovar)?
•	Is my sample mixed (with another Salmonella Typhimurium)?

I think there are probably better tools to do all of this then what I implemented at PHE 7-8 years ago but that’s the premise!

If you want to get more info from PHE/UKHSA I suggest you reach out to Anais Painset who is the lead bioinformatician for GI now.

All the best
Tim
